### ISSUE TITLE
create pygeoapi plugin

### ISSUE THREAD

**Issue by tomkralidis** [wmo-im/synop2bufr#32]
Implement pygeoapi plugin in the same pattern as implemented in [csv2bufr](https://github.com/wmo-im/csv2bufr/blob/main/csv2bufr/pygeoapi_plugin.py).

**Comment by yanbodiaosou** [comment]
I want to deal with hourly data of SYNOP to BUFR, synop2bufr can do this? Please reply as soon as possible

**Comment by RoryPTB** [comment]
Yes synop2bufr is intended to be able to handle hourly SYNOP data. Give it a try and let me know how it goes.
